content/tutorials/portenta-h7/por-ard-lvgl/content.md

```diff
@@ -78,10 +72,19 @@ we are going to work on the **Portenta_lvgl** example , The Portenta H7 comes wi
    lv_obj_align(myCustomLabel, NULL, LV_ALIGN_CENTER, 0, -40);   //We move it to the center of the screen below the 'Hello world' and align centered
    lv_label_set_text(myCustomLabel , "Custom Button");          //We set the default text
    ```
-
+   
    ![por_ard_lvgl_setup_widgets](assets/por_ard_lvgl_setup_widgets.png)
+   
+   Now that we have configured all our widgets we need to update the output by using `lv_task_handler()` `inside the loop()`
+   
+   ```cpp
+   void loop() {
+      // put your main code here, to run repeatedly:
+      lv_task_handler();
+   }
+   ```
 
-   Compile and Upload the sketch to your board. Connect the board to the USB hub with the Monitor attached to the HDMI port.  Once you power up the HUB, you will see a button with a label **CustomLabel**.
+   Compile and Upload the sketch to your board. Connect the board to the USB hub with the Monitor attached to the HDMI port.  Once you power up the HUB, you will see a button with a label saying **Custom Button**.
```

Comment:
I would move this paragraph to step 5 which will have the following substeps / flow of steps in the following order  
1. Upload sketch to board 
2. Disconnect board 
3. Connect the board to hub
4. Connect the monitor to hub
5. Power up the hub
6. See the display